scripts/install.sh

```diff
@@ -10,21 +10,13 @@ COVERAGE_POOL_PATH=$(realpath $(dirname $0)/../)
 
 # Defaults, can be overwritten by env variables/input parameters
 NETWORK_DEFAULT="development"
```

Comment:
how about adding somewhere at the top a verification for a node version? I was on 11.5.0 and it failed on `yarn install` and we can check it at the top. 12.8.1 is just an example that I happen to have, not sure what is the lowest required though
```
NODE_CURRENT_VER="$(node --version)"
NODE_REQUIRED_VER="v12.8.1"

if [ "$(printf '%s\n' "$NODE_REQUIRED_VER" "$NODE_CURRENT_VER" | sort -V | head -n1)" != "$NODE_REQUIRED_VER" ]; 
then
      echo "Required node version must be at least ${NODE_REQUIRED_VER}" 
      exit 1
fi
```

Reply:
I added `engines` property to `package.json` in e5b4f7c09028c25f99e8d8f477c3ac80d7bdfe35. It should fail the execution:

```
➜  coverage-pools git:(deployment-external-contracts) ✗ yarn install
yarn install v1.17.3
[1/5] 🔍  Validating package.json...
error @keep-network/coverage-pools@0.0.1: The engine "node" is incompatible with this module. Expected version ">= 12.0.0". Got "11.15.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Reply:
nice!